src/DBQuerys.java

```diff
@@ -0,0 +1,195 @@
+package src;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.text.SimpleDateFormat;
+
+
+
+import java.util.Date;
+import java.util.Scanner;
+public class DBQuerys {
+    SimpleDateFormat sdf = new SimpleDateFormat("dd/MM/yyyy");
+    PreparedStatement st;
+    PreparedStatement insertHotel;
+    ResultSet rs;
+    Statement stmt;
+    Scanner sc = new Scanner(System.in);
+    
+
+    public DBQuerys() {
+    }
+
+    //realiza o cadastro de novos hospedes
+    public void cadastroHospede(Connection conn) {
+        try {
+            
+                Date checkin = new Date();
+                stmt = conn.createStatement();
+                
+                System.out.println("Insira os dados do hospede");
+                System.out.print("Nome: ");
+                String name = sc.nextLine();
+                System.out.print("Email: ");
+                String emaill = sc.nextLine();
+                System.out.print("Telefone: ");
+                Long telefon = sc.nextLong();
+                String sql = "INSERT INTO hospedes (Nome, Email, Telefone) VALUES ( '" + name + "','" + emaill + "' , '"
+                        + telefon + "')";
+
+                System.out.println("Selecione o quarto pelo numero:");
+                
+                listar("quartos");
+
+
+                System.out.print("Digite o quarto: ");
+                int idQuarto = sc.nextInt();
+
+                
+                System.out.println("Digite o final da estadia dd/mm/yyyy: ");
+                String checkout = sc.next();
+                
+                stmt.executeUpdate(sql, Statement.RETURN_GENERATED_KEYS);
```

Comment:
```suggestion
               array generatedkeys = stmt.executeUpdate(sql, Statement.RETURN_GENERATED_KEYS);
               
               idHospede = generatedKeys[1]
```

se não funcionar pode ignorar